.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,17 +6,35 @@ labels: ''
 assignees: ''
 ---
 
-## Describe the bug
+## Desktop (please complete the following information)
+
+- OS: [e.g. mac M2 Sonoma, Linux Ubuntu 24.10]
```

Comment:
Here you are combining OS(sonoma) and arch(M2), however for ubuntu we don't put arch in the placeholder. Can we make it clear that we need both?